platform/docs/docs/migration-guide.md

```diff
@@ -0,0 +1,492 @@
+
+However, in OHIF v3, there have been significant changes. The rendering and tooling logic has been migrated to a new library called [`Cornerstone3D`](https://github.com/cornerstonejs/cornerstone3D-beta/). This means that all viewport rendering and tool functionalities are now handled by Cornerstone3D.
+
+Additionally, in OHIF v3, the native support for 3D functionalities previously provided by the `vtk` extension has been integrated into Cornerstone3D. As a result, there is no longer a need to use the vtk extension.
+
+To migrate from OHIF v2 to OHIF v3:
+
+- Update your code to use Cornerstone3D for viewport rendering and tools. If you don't have any custom tools or commands, you most likely won't need to make any changes.
+- If you have custom tools in `cornerstone-tools`, you will need to migrate them to Cornerstone3D. We have migrated various tools from `cornerstone-tools` to Cornerstone3D and you can
+use them as a reference (the main difference for tools are that now all toolData are stored in the world coordinate system).
+- Remove any dependencies on `react-cornerstone-viewport`, `cornerstone-tools`, and `cornerstone-core`.
+- Replace any usages of 3D functionalities from the vtk extension with the native 3D support provided by Cornerstone3D using its `VolumeViewport` component.
+
+By following these steps, you can leverage the improved rendering and tooling capabilities of Cornerstone3D and eliminate the need for the vtk extension in OHIF v3.
+
+### DICOM Segmentation & DICOM RT
+
+In OHIF v3, the equivalent extensions for RT and SEG exists with similar logic, but with various bug fixes and improvements.
+Additionally, OHIF v3 introduces new functionalities with the SEG Viewport and RT Viewport.
+
+When loading a series that contains SEG (Segmentation) or RT (RT Structure Set) data, the viewport will automatically
+switch to the corresponding SEG or RT viewport. The user will then be prompted to decide whether to load the segmentation
+or RT structure set into the viewer. This new feature addresses a common use case in which there are multiple segmentation
+series in a study, and the user only wants to load specific ones. In OHIF v3, the Segmentations are all loaded
+as 3D volumes and as a result a volume viewport is used to display them. (Stack Segmentation in Cornerstone3D is still a
+work in progress.)
+
+In OHIF v2, the user had to load all the segmentation series and then manually delete the ones they didn't want to see.
+However, in OHIF v3, the user has more control. The temporary SEG or RT viewport does not immediately load (hydrate)
+the segmentation or RT structure set. Instead, the user can decide which ones to load, reducing unnecessary
+loading and providing a more efficient workflow.
+
+This enhancement in OHIF v3 allows users to selectively load specific segmentations or RT structure sets,
```

Comment:
The new dialogs to load the segmentation to all viewports and the new SR-specific viewports are quite a big change in v3 compared to v2. Now things are loaded using frame of reference by default and this is good to explain.